react-charts/area-chart/step4.md

```diff
@@ -1,61 +1,61 @@
-Now we'll add multiple datasets to the area chart...
+In this step you will add tooltips to the area chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
-&lt;ChartArea
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-  ]} 
-/&gt;
+&lt;Chart
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) <strong>Add a `containerComponent` property to that section</strong>
+
+The `containerComponent` property takes a component instance which will be used to create a container element for standalone charts.
+
+You will add the `ChartVoronoiContainer` component which adds the ability to associate a mouse position with the data point(s) closest to it. This is useful for adding a tooltip.

```

Comment:
I need some clarification. During the intro, the future tense would be acceptable, correct? And present tense within each step?

Reply:
Future tense should be okay with the intros since we are telling them something they'll do in the future! Sorry if some present tense suggestions snuck in for intros!